### ISSUE TITLE
[Suggestion] Configurable fishing capsule-loot chance

### ISSUE THREAD

**Issue by Luke616** [Lythom/capsule#4]
Title, I've noticed that every time a treasure type item is generated while fishing , a capsule is also generated. Do you think it would be possible to make a configurable chance of this occurring instead, or a config option such that fishing loot can be disabled entirely (but chest/dungeon capsule loot is fine)? 

In the pack I am working on, fishing loot gives capsules far too frequently for my liking, especially at high luck. I would like to tone this down without disabling dungeon loot at the same time.

Thank you for your time!

**Comment by Lythom** [comment]
Thank you for your feedback !
I was not aware fishing would grant capsules. Capsule loots are not designed to be acquired in large quantities so I'll be disabling capsules as fishing loot.
I plan a new version next week, will try to include this fix as well.

What version of minecraft is your pack for (1.10, 1.11, 1.12) ?

**Comment by Luke616** [comment]
The MC version would be 1.12.2, and the Capsule version I am currently using 3.1.57. [Here ](https://minecraft.curseforge.com/projects/magiculture-2/files/2534841) (scroll to the bottom) is the modlist. Thank you for this!

**Comment by Lythom** [comment]
Hi !

The loot tables eligible to give reward capsules can now be fined tuned in capsule.cfg from version 1.12.2-3.1.63 available at https://minecraft.curseforge.com/projects/capsule/files/2537805. A new entry in the config file (lootTablesList) allows to configure where reward capsule will spawn. The loot table "gameplay/fishing/treasure" is removed from default loot tables.

I checked Magiculture+2-0.2.0.1, good pick on disabling capsules loot totally by emptying lootTemplatesPaths in capsule.cfg !

With 1.12.2-3.1.63, some possibilities to further extend your pack to your liking : 
- re-enabling capsule loots from the config (and remove/add from generated config lootTablesList the loot tables you don't want capsules in)
```
    S:lootTemplatesPaths <
        config/capsule/loot/common
        config/capsule/loot/uncommon
        config/capsule/loot/rare
     >
```
- Having one-use capsule as reward from quest book
https://github.com/Lythom/capsule/wiki/Modpack-making#create-a-reward-capsule
- Adding your own rewards in the config\capsule\loot\<rarity> for player to find in chests
https://github.com/Lythom/capsule/wiki/Modpack-making#create-a-template-to-be-used-as-loot

Let me now if you have any question, 
Good Luck !

Fixed by commit : e1a4948d2ff015b51cbf08e8c334e9ae0460a52e
